### PR TITLE
Ana/203 fix cannot read property denom of null

### LINKED ISSUE #203: Getting balances failed: cannot read property 'denom' of null
image

### PR DESCRIPTION
**Pull request by Bitcoinera**
Closes #203 

Also fixed in #198 (but here I add filtering out `null` rewards)

### CODE REVIEW

**Comment by Bitcoinera** [comment]
ouch, linter is becoming more and more picky :facepalm:  https://github.com/luniehq/lunie-light/pull/204/checks?check_run_id=1407088659

**Comment by Bitcoinera** [comment]
The fix can be found in `cosmos-reducers.js`. The rest is just linting due to the eslint upgrade

File: components/common/Address.vue [hunk 1]
```diff
@@ -17,7 +17,7 @@
 import { formatAddress } from '~/common/address'
 
 export default {
-  name: `lunie-address`,
+  name: `LunieAddress`,
```
Comment (jbibla) [inline components/common/Address.vue:20]:
```suggestion
  name: `LunieAddress`,
```

File: pages/create/index.vue [hunk 2]
```diff
@@ -26,8 +26,9 @@ import network from '~/common/network'
 const steps = [`Name`, `Password`, `Backup`, `Confirm`]
 
 export default {
-  name: `sign-up`,
+  name: `SignUp`,
```
Comment (jbibla) [inline pages/create/index.vue:29]:
```suggestion
  name: `Create`,
```

File: pages/explore/index.vue [hunk 3]
```diff
@@ -34,7 +34,7 @@ import { decodeB32 } from '~/common/address'
 import network from '~/common/network'
 
 export default {
-  name: `session-explore`,
+  name: `SessionExplore`,
```
Comment (jbibla) [inline pages/explore/index.vue:37]:
```suggestion
  name: `Explore`,
```

File: pages/proposals/index.vue [hunk 4]
```diff
@@ -70,7 +70,7 @@ import { mapState } from 'vuex'
 import network from '~/common/network'
 
 export default {
-  name: `governance-overview`,
+  name: `GovernanceOverview`,
```
Comment (jbibla) [inline pages/proposals/index.vue:73]:
```suggestion
  name: `Proposals`,
```

File: pages/recover/index.vue [hunk 5]
```diff
@@ -23,8 +23,9 @@ import network from '~/common/network'
 import { getHDPath } from '~/common/hdpath'
 
 export default {
-  name: `recover`,
+  name: `Recover`,
```
Comment (jbibla) [inline pages/recover/index.vue:26]:
```suggestion
  name: `SessionRecover`,
```

File: pages/signin/index.vue [hunk 6]
```diff
@@ -55,8 +55,9 @@ import { required, minLength } from 'vuelidate/lib/validators'
 import { getWallet, getWalletIndex } from '~/common/keystore'
 
 export default {
-  name: `sign-in`,
+  name: `SignIn`,
```
Comment (jbibla) [inline pages/signin/index.vue:58]:
```suggestion
  name: `SessionSignIn`,
```

**Pull request merged**